src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java

```diff
@@ -141,8 +142,8 @@ private void validateInformations(Technology technology) throws BadRequestExcept
       throw new BadRequestException(ValidationMessageEnums.TECHNOLOGY_ID_CANNOT_BLANK.message());
     } else if (technology.getName() == null || technology.getName().equals("")) {
       throw new BadRequestException(ValidationMessageEnums.TECHNOLOGY_NAME_CANNOT_BLANK.message());
-    } else
-      if (technology.getShortDescription() == null || technology.getShortDescription().equals("")) {
+    } else if (technology.getShortDescription() == null
+        || technology.getShortDescription().equals("")) {
```

Comment:
@jfmedeiros1820 Prefer using "".equals(technology.getShortDescription())
Using this you avoid a NullPointerException